pycytominer/cyto_utils/cells.py

```diff
@@ -445,7 +449,10 @@ def load_compartment(self, compartment):
 
         # Get data useful to pre-alloc memory
         num_cells = self.count_sql_table_rows(compartment)
-        meta_cols, feat_cols = self.get_sql_table_col_names(compartment)
+        col_names = self.get_sql_table_col_names(compartment)
+        if self.features != "infer":  # allow to get only some features
+            col_names = [x for x in col_names if x in self.features]
+        meta_cols, feat_cols = self.split_columns_into_classes(col_names)
```

Comment:
```suggestion
        meta_cols, feat_cols = self.split_column_categories(col_names)
```

To avoid the name collision with pythonic "classes". (Maybe a different name is better than my proposal)